tools/modules/software/module_netbox.sh

```
@@ -0,0 +1,164 @@
+	["module_netbox,doc_link"]="https://netbox.readthedocs.io/en/stable/"
+	["module_netbox,group"]="Management"
+	["module_netbox,port"]="8222"
+	["module_netbox,arch"]="x86-64 arm64"
+)
+
+#
+# Module netbox
+#
+function module_netbox () {
+	local title="netbox"
+	local condition=$(which "$title" 2>/dev/null)
```

Shellcheck (0.10.0)

[warning] condition appears unused. Verify use (or export if used externally).

(SC2034)